pkg/controller/core/resourceflavor_controller.go

```diff
@@ -68,14 +68,7 @@ func (r *ResourceFlavorReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	ctx = ctrl.LoggerInto(ctx, log)
 	log.V(2).Info("Reconciling ResourceFlavor")
 
-	if flavor.ObjectMeta.DeletionTimestamp.IsZero() {
```

Comment:
I would keep this, so that we can add a finalizer to resources created in version 0.1